client/a8c-for-agencies/sections/partner-directory/agency-details/hooks/use-submit-form.ts

```diff
@@ -19,7 +19,10 @@ const getImageFile = async ( agencyId: number, blobURL: string ) => {
 
 export default function useSubmitForm( { formData, onSubmitSuccess, onSubmitError }: Props ) {
 	const agencyId = useSelector( getActiveAgencyId );
-	const addMedia = useAddMedia();
+	const addMedia = useAddMedia( {
```

Comment:
All things being equal, this doesn't seems like the right level to place such an abstraction. I'd expect the path/namespace are not exposed to the external `useAddMedia` consumer.